src/pages/DashboardPage.tsx

```diff
@@ -1,12 +1,47 @@
-import React from "react";
+import React, { Component } from "react";
 import Layout from "../components/Layout";
+import Context from "../store/Context";
 
-const DashboardPage = () => {
-  return (
-    <div>
-      <Layout />
-    </div>
-  );
-};
+class DashboardPage extends Component {
+  static contextType = Context;
+  // For TS pre-3.7:
+  context!: React.ContextType<typeof Context>;
+  // For TS 3.7 and above:
+  // declare context: React.ContextType<typeof MyContext>;
+  ignoreLastFetch = false;
+
+  componentDidMount() {
+    // Fetch data initially
+    this.fetchOrgsAndUser();
+  }
+
+  // componentDidUpdate(prevProps: any) {
+  //   const oldId = prevProps.params.invoiceId;
+  //   const newId = this.props.params.invoiceId;
+  //   if (newId !== oldId) this.fetchInvoice();
+  // }
+
+  componentWillUnmount() {
+    // Allows us to ignore an inflight request
+    this.ignoreLastFetch = true;
+  }
+
+  fetchOrgsAndUser() {
```

Comment:
Doing this in this top-level dashboard page so that _all_ its children also get updated when these things change. I _think_ for now, only orgs and username are the main things we'll need to update upon landing here. Anything else that changes will be due to UI interactions, so easier to capture from sub-components.